wizard.sh

```diff
@@ -433,13 +433,6 @@ main() {
         fi
         log_info "installing crowdsec"
         install_crowdsec
-        # lapi register
-        MACHINE_ID=""
-        if [[ ${DOCKER_MODE} == "false" ]]; then
-            ${CSCLI_BIN_INSTALLED} machines add --force "$(cat /etc/machine-id)" --password "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)" -f "${CROWDSEC_CONFIG_PATH}/${CLIENT_SECRETS}"
-        fi
-            
-        log_info "Crowdsec LAPI registered"
```

Comment:
Why get rid of this ?

Reply:
The official one is that: `--bininstall` doesn't have to register to LAPI. The less official one is that `$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)` caused the docker image of CI to hang

Reply:
ok then